### PR TITLE
Add audio modding article

### PR DESCRIPTION
**Pull request by milleroski**
Hi! I finally decided to port my audio modding guide over to this wiki! It extensively explains how someone can create an audio replacement mod using CustomSounds from the beginning to the end.

Please let me know if I should change anything -- I'm always open to suggestions :)

### CODE REVIEW

**Comment by MaxWasUnavailable** [comment]
We'll probably want to set up a custom category for mod-specific guides (= guides using other mods or APIs to implement something, rather than being completely standalone). We will merge this once we have that set up.

Matter of keeping the playing field fair for all APIs.

**Comment by milleroski** [comment]
Yeah, I totally agree with you there. It would make navigation for users easier as well since it's not immediately obvious that my guide uses CustomSounds -- that's why I wrote a note at the beginning in the first place.

**Comment by legoandmars** [comment]
Appreciate the PR a ton, should make helping people make audio mods much easier

Once #29 is merged in I'll look into setting up a mod-specific section so we can work on getting this in

File: docs/user-guide/modding/audio-modding-customsounds.md
Comment (legoandmars) [inline docs/user-guide/modding/audio-modding-customsounds.md:3]:
```suggestion
::: warning 
This is not a tutorial on how to replace audio with BepInEx using Harmony Patching or with LCSoundTool. This is a guide for using CustomSounds to replace audio. If you want to make a standalone audio mod without CustomSounds, this guide is not for you.
:::
```

replace with new formatting

File: docs/user-guide/modding/audio-modding-customsounds.md
Comment (legoandmars) [inline docs/user-guide/modding/audio-modding-customsounds.md:38]:
```suggestion
::: tip
If you already have found the audio file you're looking for, you can skip directly to the next section.
:::
```
replace with new formatting

File: docs/user-guide/modding/audio-modding-customsounds.md
Comment (legoandmars) [inline docs/user-guide/modding/audio-modding-customsounds.md:21]: I wouldn't recommend AssetStudio, it only supports up to 2022.3.1

Just AssetRipper should work fine

**Review by legoandmars — CHANGES_REQUESTED** [review]
Thanks a ton for the PR, the article looks good, we've just been undergoing a lot of formatting changes and the formatting needs to be slightly updated

- Any changes to sidebar.md and index.html can be discarded, as we're using a new structure now
- All images should be moved to `/docs/public/images/customsounds` with the in-guide links changed to `/images/customsounds/IMAGENAME`
- The page should  be moved to `/docs/apis/apis/customsounds.md`
- The page can be added to the sidebar through `docs/.vitepress/config.mts`, under the "Modding APIs" section (right under the page that says "Modding APIs Overview"

**Comment by milleroski** [comment]
Alright, I've adapted the guide to the new layout and added the changes you've requested, so everything should work fine now. I've tested it locally by running "npm run dev" and it looks fine, but it seems that the workflow is failing to configure the github pages. Specifically, I get the error "Create pages site failed". Any ideas as to why?

**Comment by legoandmars** [comment]
> Alright, I've adapted the guide to the new layout and added the changes you've requested, so everything should work fine now. I've tested it locally by running "npm run dev" and it looks fine, but it seems that the workflow is failing to configure the github pages. Specifically, I get the error "Create pages site failed". Any ideas as to why? Additionally, the guide should be probably referenced directly in the API overview page and given a gold star.

I haven't messed with how the github pages build works on forks, it's probably just not set up properly by default. I doubt it'll be an issue

**Review by legoandmars — APPROVED** [review]

**Pull request merged**
